docs/fileset/README.md

```diff
@@ -160,22 +212,39 @@ Updating a fileset works the same way as a dataset.
 Updating a fileset record with a new file is not supported. Only the metadata can be changed. If you need to upload a new file, please create a new record. 
 :::
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 jfs.fileset("fileset_name")  
  .update({ "isDefaultImage": false })  
  .where(field => field("name").isEqualTo("companyLogo.png"))
- .subscribe();  
+ .subscribe(data=>{}, err=>{});  
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
I will work on this in the next update.